shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -272,6 +286,21 @@ - (void)loadView {
   self.view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
 
   [view release];
+
+  // Show the launch screen view again on top of the FlutterView if available.
+  // This launch screen view will be removed once the first Flutter frame is rendered.
+  NSString* launchStoryboardName =
```

Comment:
The line below should keep it from doing anything if the view controller is being pushed or presented by another